docs/hive_metrics.md

```diff
@@ -127,4 +128,20 @@ Some of these metrics are optional and the admin can opt for logging them via `H
 |               hive_controller_reconcile_seconds                |           N            |    N     |
 |             hive_cluster_deployment_syncset_paused             |           N            |    N     |
 |       hive_cluster_deployment_provision_underway_seconds       |           N            |    N     |
-|  hive_cluster_deployment_provision_underway_install_restarts   |           N            |    N     |
\ No newline at end of file
+|  hive_cluster_deployment_provision_underway_install_restarts   |           N            |    N     |
+
+### Example: Configure metricsConfig
+
+```sh
+oc edit hiveconfig -n hive
+```
+
+```yaml
+spec:
+  metricsConfig:
+    metricsWithDuration:
+      - name: currentClusterSyncFailing
+        duration: 1s
```

Comment:
I know this is just an example, but maybe set the duration to 1 hour or larger instead. One of the reasons we added this duration field was to enable admins to not overload their prometheus and only logging the metrics for cases they care about or want to alert on.